### DISCUSSION TITLE
OTA not working - self signed certificates

### DISCUSSION THREAD

**Discussion by keescyclingcroissant** [Koenkk/zigbee2mqtt#20423] [Devices]
Hi there!

Currently, I'm experiencing some issues with OTA in Z2M running as a Home Assistant add-on.
After a succesfull installation and including devices in the Zigbee network, the devices got a notification that a firmware upgrade is available.

However, every time I hit the 'update device firmware' button, I get the following error message: 'Update of 'DEVICE NAME' failed (self signed certificate in certificate chain)'. 

Already searched for this error on the web, but couldn't find anything related.

Anybody else also facing this issue?

Thnx

**Comment by BastianKa** [comment]
Hey @casvr97,

same issue here.
Yesterday I successfully updated two Philips Hue Play Lightbars via OTA.
Today I tried to update a third Hue Play Lightbar and got the same error in the logs: "**Update of 'Hue_Play_Lightbar_3' failed (self-signed certificate in certificate chain)**".

Best regards

**Reply by keescyclingcroissant** [reply]
@BastianKa weird but interesting. I’m also getting the errors on Hue products…

After multiple tries I sometimes wont get the error, the log mentions start update, but no progress bar or anything. The next error is that the device didn’t respond to the OTA request.

Do you experience the same?

**Reply by BastianKa** [reply]
No, my logs only shows the error as described. Even after several attempts.

**Comment by uphillbattle** [comment]
I see the same. I have opened an issue #20432 referencing this discussion.

**Reply by keescyclingcroissant** [reply]
@uphillbattle nice! I see that isue is ‘closed’ however I’m still experiencing the same issue (even after recent Z2M’s update). Did you had any update on the issue?

**Reply by uphillbattle** [reply]
No, I think the OTA fix hasn’t been implemented yet.

**Reply by uphillbattle** [reply]
With the latest release 1.35.1, OTA for Philips Hue works 👍

**Reply by keescyclingcroissant** [reply]
Yes indeed, can confirm aswell!
